fire/cli/mtl.py

```diff
@@ -107,59 +109,24 @@ def get_observation_strings(
                     ]
                     observationer.append(reordered)
         except FileNotFoundError:
-            fire.cli.print(f'Kunne ikke læse filen "{filnavn}"')
+            fire.cli.print(f"Kunne ikke læse filen '{filnavn}''")
     return observationer
 
 
 # ------------------------------------------------------------------------------
-def punkt_information(ident: str) -> PunktInformation:
-    """Find alle informationer for et fikspunkt"""
-    pi = aliased(PunktInformation)
-    pit = aliased(PunktInformationType)
-    try:
-        punktinfo = (
-            fire.cli.firedb.session.query(pi)
-            .filter(pit.name.startswith("IDENT:"), pi.tekst == ident)
-            .first()
-        )
-    except NoResultFound:
-        fire.cli.print(f"Error! {ident} not found!", fg="red", err=True)
-        sys.exit(1)
-    if punktinfo is not None:
-        fire.cli.print(f"Fandt {ident}", fg="green", err=False)
-    else:
-        fire.cli.print(f"Fandt ikke {ident}", fg="cyan", err=False)
-    return punktinfo
-
-
-# ------------------------------------------------------------------------------
-def punkt_kote(punktinfo: PunktInformation, koteid: int) -> Koordinat:
-    """Find aktuelle koordinatværdi for koordinattypen koteid"""
-    if punktinfo is None:
-        return None
-    for koord in punktinfo.punkt.koordinater:
-        if koord.sridid != koteid:
-            continue
-        if koord.registreringtil is None:
-            return koord
-    return None
-
-
-# ------------------------------------------------------------------------------
-def punkt_geometri(punktinfo: PunktInformation, ident: str) -> Tuple[float, float]:
+# Kunne måske være en property på Punktobjektet
+def punkt_geometri(punkt: Punkt) -> Tuple[float, float]:
     """Find placeringskoordinat for punkt"""
-    if punktinfo is None:
-        return (11, 56)
     try:
-        geom = fire.cli.firedb.hent_geometri_objekt(punktinfo.punktid)
+        geom = firedb.hent_geometri_objekt(punkt.id)
         # Turn the string "POINT (lon lat)" into the tuple "(lon, lat)"
-        geo = eval(str(geom.geometri).lstrip("POINT").strip().replace(" ", ","))
-        # TODO: Perhaps just return (56,11) Kattegat pain instead
-        assert len(geo) == 2, "Bad geometry format: " + str(geom.geometri)
+        return eval(str(geom.geometri).lstrip("POINT").strip().replace(" ", ","))
     except NoResultFound:
-        fire.cli.print(f"Error! Geometry for {ident} not found!", fg="red", err=True)
+        fire.cli.print(
+            f"Error! Geometry for {punkt.ident} not found!", fg="red", err=True
+        )
         sys.exit(1)
-    return geo
+    return (11, 56)
```

Comment:
Det er på ingen måde åbenlyst, men placeringskoordinaten *er* direkte tilgængelig fra et Punkt:

```
punkt.geometriobjekter[-1].geometri.__geo_interface__["coordinates"]
```

Det er ikke kønt, men mindre kompliceret end den funktion du har lavet her. Jeg tror en to-trins raket af properties kan løse det endnu bedre:

```
class Punkt():
    ...

    @property
    def geometri(self):
        return self.geometriobjekter[-1]
```

samt

```
class Geometri():
    ...
  
    @property
    def koordinater(self):
        return self.geometri.__geo_interface__["coordinates"]
```

hvorefter man så vil kunne sige

```
punkt.geometri.koordinater
```

Det synes jeg er en god løsning.

Reply:
Jeg var noget i tvivl om hvorvidt man altid kunne være sikker på at geometriobjekter[-1] også var identisk med den sidst skrevne, men jeg har set at det vist også er denne logik der benyttes forskellige steder i API-koden. Jeg prøver lige at indføre din løsning (det bliver nok først på mandag)

Reply:
> Jeg var noget i tvivl om hvorvidt man altid kunne være sikker på at geometriobjekter[-1] også var identisk med den sidst skrevne,

Jo, det er garanteret, da der sorteres efter `objectid`jf. denne her stump fra `Punkt`
:
```
geometriobjekter = relationship(
        "GeometriObjekt", order_by="GeometriObjekt.objectid", back_populates="punkt"
    )
```

Reply:
Jeg synes også det er en god løsning, men har spekuleret mig grøn og gul over den rette rækkefølge at udføre dette på. Er endt på at jeg lige nu indfører din [-1] løsning i MTL, og så laver en separat PR for `@property` implementeringen